test/utils/shippable/windows.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -eux
+
+set -o pipefail
+
+declare -a args
+IFS='/:' read -ra args <<< "${TEST}"
+
+target="windows/ci/group${args[1]}/"
+
+ansible-test integration --explain 2>&1 | grep ' integration: ' | grep -v ' integration: none$' > /tmp/posix.txt
+
+if [ -s /tmp/posix.txt ]; then
+    echo "Detected changes requiring integration tests which are not specific to Windows:"
```

Comment:
The logic for this test is actually wrong. I'll add proper comments once I've fixed the logic.